src/plugins/ResizeReorder/ReorderHandle.js

```diff
@@ -0,0 +1,293 @@
+/**
```

Comment:
ReorderHandle.js -> Logic for handling reordering. It calculates the displacement of reordering cell and passed is up to user.